Formula/newsboat.rb

```diff
@@ -5,12 +5,7 @@ class Newsboat < Formula
   sha256 "82d5e3e2a6dab845aac0bf72580f46c2756375d49214905a627284e5bc32e327"
   head "https://github.com/newsboat/newsboat.git"
 
-  bottle do
```

Comment:
The bottle block shouldn't be modified or deleted manually. The test-bot handles it.

Reply:
Oh I see it's because I started from my previous branch in which the bottle wasn't added